data-es2016plus.js

```diff
@@ -734,8 +734,10 @@ exports.tests = [
          && Object.prototype.__lookupGetter__.call(obj, "qux") === undefined
          && Object.prototype.__lookupGetter__.call(obj, "baz") === undefined;
          */},
-        res: temp.basicDefineGetterResults,
-      },
+        res: Object.assign({}, temp.basicDefineGetterResults, {
+          firefox2:   true,
```

Comment:
Funny. FF2-3.x missed in `basicDefineGetterResults` because hasn't support `Object.getOwnPropertyDescriptor`.